packages/@aws-cdk/cdk/test/core/test.tag-manager.ts

```diff
@@ -4,34 +4,34 @@ import { TagManager } from '../../lib/core/tag-manager';
 
 export = {
   '#setTag() supports setting a tag regardless of Type'(test: Test) {
-    const notTaggable = new TagManager(TagType.NotTaggable);
+    const notTaggable = new TagManager(TagType.NotTaggable, '');
```

Comment:
put something like "AWS::Resource::Type" instead of empty strings so it's clearer what this string is